cluster-script/helpers.yaml

```diff
@@ -79,9 +79,16 @@ data:
 
     # Prints CPU manufacturer / Model name & revision
     function get_model() {
-        lscpu | awk '/^Vendor ID:/ {$1=""; $2=""; gsub(/^ +/,""); gsub(/ +$/,""); s=s "" $0}
-                     /^Model name:/ {$1=""; $2=""; gsub(/^ +/,""); gsub(/ +$/,""); s=s " " $0}
-                     END {print s}'
+        lscpu | awk '/^Vendor ID:/  { $1=""; $2="";
+                                        gsub(/^ +/,"" );
+                                        gsub(/ +$/,"");
+                                        s=$0 }
+                     /^Model name:/ { $1=""; $2="";
+                                        gsub(/^ +/,"");
+                                        gsub(/ +$/,"");
+                                        s=s " " $0 }
```

Comment:
Should we also include the `Model` and `Stepping` numbers in the output of `get_model`? Then we can replace APM X-Gene with eMAG maybe more reliably.

Reply:
This is more a note for later, does not block it now.

Reply:
I was planning to work with Ampere to get that fixed and upstreamed to `util-linux` directly. Not sure yet how to tell X-Gene from eMAG yet, maybe Ampere can help.